src/IO/AbstractIOHandlerHelper.cpp

```diff
@@ -36,9 +39,15 @@ namespace openPMD
         std::string path,
         AccessType accessTypeBackend,
         Format format,
-        MPI_Comm comm
-    )
+        MPI_Comm comm,
+        std::string const & options )
     {
+#   if openPMD_HAVE_JSON
```

Comment:
we always have `openPMD_HAVE_JSON`

Reply:
All traces of this should be removed from the source tree now, according to `grep`.